cardano-testnet/src/Testnet/Process/Run.hs

```diff
@@ -313,6 +313,7 @@ data ExecutableError
   | RetrievePlanJsonFailure IOException
   | ReadFileFailure IOException
   | MissingExecutable FilePath String
+  | CannotFindExecutable String
```

Comment:
What's the difference between these two?

Reply:
@carbolymer> I added a commit to provide better names and Haddock :+1: